plumgrid_playbooks/enable_pg_extensions.yml

```diff
@@ -19,16 +19,9 @@
   tasks:
     - name: Remove previous PLUMgrid sources.list
       file:
-        path: /etc/apt/sources.list.d/plumgrid.list
+        path: "/etc/apt/sources.list.d/plumgrid.list"
         state: absent
 
-    - name: Add PLUMgrid openstack to repo
-      lineinfile:
-        dest: /etc/apt/sources.list.d/plumgrid.list
-        line: "deb {{ plumgrid_repo }}/openstack/deb/{{ openstack_version }} openstack-{{ openstack_version }} {{ plumgrid_component }}"
-        state: present
-        create: yes
-
     - name: Copy PLUMgrid GPG-key file
```

Comment:
@JaveriaK instead of copying the GPG key to the server and then add it using can simply upload the public part of your GPG key to the public GPG key server like `keyserver.ubuntu.com` and then directly get it from there like this:

```yaml
- name: Add Plumgrid PPA apt key
  apt_key:
    id: "AA798760"
    keyserver: "keyserver.ubuntu.com"
    state: present
```